File: tests/test_data/ChatGPT/v1/Transcribed/Factorial.py
def calcular_factorial(numero):
    factorial = 1
    for i in range(1, numero + 1):
        factorial *= i
    return factorial

def main():
    try:
        numero = int(input("Ingrese un número: "))
        factorial = calcular_factorial(numero)
        print(f"El factorial de {numero} es: {factorial}")
    except ValueError:
        print("Ingrese un número válido")

if __name__ == "__main__":
    main()

File: tests/test_data/ChatGPT/v1/Transcribed_JS/Factorial.js
const leerNumero = async () => {
  const respuesta = await prompt("Ingrese un número: ");
  return parseInt(respuesta);
};

class Factorial {
  constructor(numero) {
    this.numero = numero;
  }

  calcularFactorial() {
    let factorial = 1;
    for (let i = 1; i <= this.numero; i++) {
      factorial *= i;
    }
    return factorial;
  }
}

const main = async () => {
  const numero = await leerNumero();
  const factorial = new Factorial(numero);
  const resultado = factorial.calcularFactorial();
  console.log(`El factorial de ${numero} es: ${resultado}`);
};

main();